onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperator.cpp

```diff
@@ -47,30 +47,43 @@
 
         if (contextPrivate->IsDmlGraphNode())
         {
-            // Create an edge list using sentinels for unused edges, as required by the SetDmlOperator ABI
-            auto ReplaceUnusedEdgeIndicesWithSentinel = [](gsl::span<const std::optional<uint32_t>> indices)
+            MLOperatorGraphDesc operatorGraphDesc = {};
+            operatorGraphDesc.nodeCount = 1;
+            std::vector<const DML_OPERATOR_DESC*> opDescs({&operatorDesc});
+            operatorGraphDesc.nodesAsOpDesc = opDescs.data();
+
+            std::vector<DML_INPUT_GRAPH_EDGE_DESC> inputEdges;
+            for (int inputIndex = 0; inputIndex < m_kernelInputIndices.size(); inputIndex++)
```

Comment:
>int [](http://example.com/codeflow?start=17&length=3)

Doesn't this give a signed/unsigned comparison mismatch when building? Other `for` loops below use `uint32_t`.

Reply:
It didn't. But I will use uint32_t